### ISSUE TITLE
Using another renderer

### ISSUE THREAD

**Issue by felipegb94** [lzqsd/InverseRenderingOfIndoorScene#2]
Hi!

How hard do you think it would be to translate the Optix Renderer scene description files to Mitsuba scene description files? 

I think it would be interesting to be able to render this dataset using Mitsuba and some of the extensions available for Mitsuba such as [MitsubaToF](https://github.com/cmu-ci-lab/MitsubaToFRenderer) (for transient rendering) and [MitsubaCLT](https://github.com/cmu-ci-lab/mitsuba_clt) (for computational light transport imaging systems), and maybe even Mitsuba 2 (for differentiable rendering).

Thanks! 
Felipe

**Comment by ttsesm** [comment]
Yes this would be quite nice, personally I would be interested to use the scene description files in Radiance but if Mitsuba files are available porting from Mitsuba to Radiance might not be a big deal.

**Comment by lzqsd** [comment]
Actually the scene description file is very similar to Mitsuba. The only different part might be the material definition. If you use lambertian materials or phong materials, probably you can directly process the scene description files with mitsuba renderer.

**Comment by felipegb94** [comment]
Sounds good! Thanks for the reply. I look forward to the release of the OpenRooms dataset, together with the scene description files. 

Since the scene description files are similar, I bet that it might be easy to write a script that just modifies the more complex materials so that they work in Mitsuba. I will probably try to render the dataset with Mitsuba once you release it.

**Comment by ttsesm** [comment]
@lzqsd for the material definition will you follow the Wavefront `.mtl` format, the Mitsuba `.xml` format or something else?
